scripts/skills/effects/rf_commanded_effect.nut

```diff
@@ -0,0 +1,15 @@
+this.rf_commanded_effect <- ::inherit("scripts/skills/skill", {
+	m = {},
+
+	function create()
+	{
+		this.m.ID = "effects.rf_commanded";
+		this.m.Name = "Commanded by an ally";
```

Comment:
For consistency I'd add skill type StatusEffect to it. As it is Hidden it won't show up, but at least it will be listed among existing status effects if someone queries that list.